plugins/ssr/ssrequality.ml

```diff
@@ -128,10 +128,9 @@ let newssrcongrtac arg ist gl =
     x, re_sig si sigma in
   let arr, gl = pf_mkSsrConst "ssr_congr_arrow" gl in
   let ssr_congr lr = EConstr.mkApp (arr, lr) in
+  let eq, gl = pf_fresh_global Coqlib.(lib_ref "core.eq.type") gl in
```

Comment:
When equality is universe polymorphic, then we need to keep the universes in context. Therefore this line has been moved up.